### PR TITLE
fixes#9334: adds more information in Step5

### LINKED ISSUE #9334: [DOCS]: add more information in Step 5 of the Open Source Roadmap
### Description

Edit:
~~##Problem~~
In my review of the roadmap, I noticed Step 5 is missing key information on contributing effectively to open source projects.

## Possible Suggestion 
Per [@SaraJaoude’s suggestion in issue #9152](https://github.com/EddieHubCommunity/BioDrop/issues/9152), it would be beneficial to mention giving code reviews as a contributing method. Something like this:

## Code Reviews 
- Ways to give effective feedback 
- Importance of etiquette when giving reviews

## Benefits 
By adding this information, beginners would: 
 - Understand that contributing to open source can be done in many ways, not just pushing commits.
 - Gain a better understanding of how to interact with others in a open source project.

### Screenshots

_No response_

### Do you want to work on this issue?

Yes

### Additional information

~~I would like to do count this towards my Hacktoberfest contribution list.~~ All contributions other than Profile creation/changes count towards Hacktoberfest.

### PR DESCRIPTION
**Pull request by Vidip-Ghosh**
## Fixes Issue

Closes #9334 

## Changes proposed

I've added more information in Open Source Roadmap. 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-10-09 at 21 44 50

### CODE REVIEW

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]
Hello! Thank you for contributing.  In reviewing your changes against the issue, it appears there may be a misunderstanding about the request.  The issue states to add information about code reviews as a way to contribute. 

Take a look and let us know if the issue is clear in what needs to be updated.

**Comment by Vidip-Ghosh** [comment]
> Hello! Thank you for contributing. In reviewing your changes against the issue, it appears there may be a misunderstanding about the request. The issue states to add information about code reviews as a way to contribute.
> 
> Take a look and let us know if the issue is clear in what needs to be updated.

Its clear, more information related to code review has to be added.

**Comment by Vidip-Ghosh** [comment]
Code review as a way to contribute can help developers understand the codebase, share feedback that helps them increase their coding skills, catch bugs, ensure no point of failure, increase collaboration among team members, etc.

**Are these points okay or shall I add more?**

**Comment by amandamartin-dev** [comment]
> Code review as a way to contribute can help developers understand the codebase, share feedback that helps them increase their coding skills, catch bugs, ensure no point of failure, increase collaboration among team members, etc.
> 
> **Are these points okay or shall I add more?**

I think this is almost there - I added one comment inline to consider.

**Review by amandamartin-dev — DISMISSED** [review]
This looks good! Thank you for considering the updates.  Another maintainer will confirm as well prior to merging. Thank you for your contributions

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Thank you.  However you have added details of how to do code contributions to the Open Source Roadmap Index page. 

The issue refers to adding code contributions to the Step 5 page as a new heading: https://www.biodrop.io/docs/open-source-roadmap/what-to-contribute

Please can you delete the wording you have added from the index page and add this as a new heading in the main body of the Step 5 page.

**Comment by amandamartin-dev** [comment]
> Thank you. However you have added details of how to do code contributions to the Open Source Roadmap Index page.
> 
> The issue refers to adding code contributions to the Step 5 page as a new heading: https://www.biodrop.io/docs/open-source-roadmap/what-to-contribute
> 
> Please can you delete the wording you have added from the index page and add this as a new heading in the main body of the Step 5 page.

Oh no! This is my fault Sara - I read the issue submitted by Christine and it did not include a link to where these changes should take place. I also thought it meant the index page.  Apologies @Vidip-Ghosh for creating extra work for you

**Comment by Vidip-Ghosh** [comment]
> > Thank you. However you have added details of how to do code contributions to the Open Source Roadmap Index page.
> > The issue refers to adding code contributions to the Step 5 page as a new heading: https://www.biodrop.io/docs/open-source-roadmap/what-to-contribute
> > Please can you delete the wording you have added from the index page and add this as a new heading in the main body of the Step 5 page.
> 
> Oh no! This is my fault Sara - I read the issue submitted by Christine and it did not include a link to where these changes should take place. I also thought it meant the index page. Apologies @Vidip-Ghosh for creating extra work for you

Can you please review now?

**Comment by Vidip-Ghosh** [comment]
@amandamartin-dev can you please review the changes? 
Sorry for tagging but due to branch getting updated day by the day, its giving conflicts.

**Comment by SaraJaoude** [comment]
> amandamartin-dev can you please review the changes? Sorry for tagging but due to branch getting updated day by the day, its giving conflicts.

You are not having conflicts in your branch because no one else is changing this file. You keep merging in main branch into your branch, which is good practice, but this will not cause conflicts - so there is no need to tag a specific maintainer.

Note: Your changes are in the wrong place in the file, I will leave a review with more information.

**Comment by Vidip-Ghosh** [comment]
> You have added the wording as a new list item under the "Common project improvements" section.
> 
> What we are looking, is for "Code Reviews" to be it's own section on this page (such as "Contributing Guide").

So shall I create a different section?

**Comment by Vidip-Ghosh** [comment]
Screenshot 2023-10-18 at 16 48 49
Like this. Am I right??

**Comment by Vidip-Ghosh** [comment]
> You have added the wording as a new list item under the "Common project improvements" section.
> 
> What we are looking, is for "Code Reviews" to be it's own section on this page (such as "Contributing Guide").

Please review the changes I've made whether it's correct or not.

**Comment by Vidip-Ghosh** [comment]
> > You have added the wording as a new list item under the "Common project improvements" section.
> > What we are looking, is for "Code Reviews" to be it's own section on this page (such as "Contributing Guide").
> 
> Please review the changes I've made whether it's correct or not.



> > You have added the wording as a new list item under the "Common project improvements" section.
> > What we are looking, is for "Code Reviews" to be it's own section on this page (such as "Contributing Guide").
> 
> Please review the changes I've made whether it's correct or not.

@SaraJaoude

File: pages/docs/open-source-roadmap/what-to-contribute.js [hunk 1]
```diff
@@ -94,6 +94,12 @@ export default function DocsContribute() {
             </span>
           </li>
         </ul>
+        <h2 className="mt-16 text-2xl font-bold tracking-tight text-primary-high dark:text-primary-low">
+          Code Reviews
+        </h2>
+        <p>
+          Code Reviews are another way to contribute as it can help developers understand the codebase, share feedback that helps them increase their coding skills, catch bugs, ensure no point of failure, increase collaboration among team members, etc.
```
Comment (SaraJaoude) [inline pages/docs/open-source-roadmap/what-to-contribute.js:101]:
```suggestion
          Code Reviews are another way to contribute as they help understand the codebase, share feedback that improves coding skills, catches bugs and increases collaboration among the community.
```

**Pull request merged**